plugins/tweak/tweak.cpp

```diff
@@ -74,6 +75,16 @@ DFhackCExport command_result plugin_init(color_ostream &out, vector<PluginComman
 
     TWEAK_HOOK("reaction-gloves", reaction_gloves_hook, produce);
 
+    TWEAK_HOOK("material-size-for-melting", material_size_for_melting_armor_hook, getMaterialSizeForMelting);
+    TWEAK_HOOK("material-size-for-melting", material_size_for_melting_gloves_hook, getMaterialSizeForMelting);
+    TWEAK_HOOK("material-size-for-melting", material_size_for_melting_shoes_hook, getMaterialSizeForMelting);
+    TWEAK_HOOK("material-size-for-melting", material_size_for_melting_helm_hook, getMaterialSizeForMelting);
+    TWEAK_HOOK("material-size-for-melting", material_size_for_melting_pants_hook, getMaterialSizeForMelting);
+    TWEAK_HOOK("material-size-for-melting", material_size_for_melting_weapon_hook, getMaterialSizeForMelting);
+    TWEAK_HOOK("material-size-for-melting", material_size_for_melting_trapcomp_hook, getMaterialSizeForMelting);
+    TWEAK_HOOK("material-size-for-melting", material_size_for_melting_tool_hook, getMaterialSizeForMelting);
+    rng.init();
```

Comment:
referring to a static from inside a header is.. sketchy. I'll see if I can suggest a better encapsulation in the .h file

Reply:
Thanks, adjusted that as you suggested.